src/Admin/CustomerPresenter.php

```diff
@@ -241,55 +259,24 @@ public function createComponentCustomers(): AdminGrid
 		}, '', 'accountsAssigned', 'Účet', ['0' => 'Bez účtu', '1' => 'S účtem'])->setPrompt('- Účet -');
 
 		$grid->addFilterDatetime(function (ICollection $source, $value): void {
-			$source->where('this.createdTs >= :createdTs_from', ['createdTs_from' => $value]);
-		}, '', 'createdTs_from', null, ['defaultHour' => '00', 'defaultMinute' => '00'])
-			->setHtmlAttribute('class', 'form-control form-control-sm flatpicker')
-			->setHtmlAttribute('placeholder', 'Registrace od');
-
-		$grid->addFilterDatetime(function (ICollection $source, $value): void {
-			$source->where('this.createdTs <= :createdTs_to', ['createdTs_to' => $value]);
-		}, '', 'createdTs_to', null, ['defaultHour' => '23', 'defaultMinute' => '59'])
-			->setHtmlAttribute('class', 'form-control form-control-sm flatpicker')
-			->setHtmlAttribute('placeholder', 'Registrace do');
-
-		$grid->addFilterDatetime(function (ICollection $source, $value): void {
-			$source->where('lastOrder.createdTs >= :lastOrder_createdTs_from', ['lastOrder_createdTs_from' => $value]);
-		}, '', 'lastOrder_createdTs_from', null, ['defaultHour' => '00', 'defaultMinute' => '00'])
+			$source->where('lastOrder.createdTs >= :created_from', ['created_from' => $value]);
+		}, '', 'date_from', null)
 			->setHtmlAttribute('class', 'form-control form-control-sm flatpicker')
-			->setHtmlAttribute('placeholder', 'Poslední obj. od');
+			->setHtmlAttribute('placeholder', 'Datum objednávky od');
```

Comment:
Uprav prosím aby tam bylo ['defaultHour' => '00', 'defaultMinute' => '00'], pro výchozí OD, pro DO zase 23:59.

je to asi chyba mergování, moje změny se tady občas ztratily